taxi/tests/test_views.py

```diff
@@ -0,0 +1,68 @@
+from django.test import TestCase
+from django.urls import reverse
+from taxi.models import Manufacturer, Car, Driver
+
+
+class ManufacturerSearchViewTest(TestCase):
+    def setUp(self):
+        self.driver = Driver.objects.create_user(
+            username="Driver_1",
+            password="Password123",
+        )
+        self.client.login(username="Driver_1", password="Password123")
+
+        Manufacturer.objects.create(name="Tesla", country="USA")
+        Manufacturer.objects.create(name="Toyota", country="Japan")
+
+    def test_search_by_name(self):
+        response = self.client.get(reverse("taxi:manufacturer-list"),
+                                   {"name": "Tesla"})

```

Comment:
Ensure that the URL name 'taxi:manufacturer-list' matches the name defined in your URL configuration for the manufacturer list view.